src/components/grid-aware/Covid19InfoBoxBlock/Covid19InfoBoxBlock.module.css

```diff
@@ -0,0 +1,33 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  background: var(--color-gray-300);
+  grid-column: 2;
+  padding: 60px;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: block;
+  }
+
+  .bleedMainContent {
+    padding: 50px 40px;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  font-size: 30px;
```

Comment:
Let's remove this, since it's already set by the `font:` shorthand above. We generally don't need to manually set the font size, font family, font weight, or line-height, since our [global font variables](https://github.com/ShelterTechSF/sheltertech.org/blob/ffe2ad112811ae214664cdc80990a9c03dd1c142/src/stylesheets/global.css#L54-L62) should already cover them.

Edit: Actually, I see why you put this here, since in mobile, we still want this to be 30px tall, but in mobile, `--font-title` is smaller. Could you instead create a media query and set the font to `--font-headline` in mobile, which has a size of 30px?

```suggestion
```

Reply:
👍 Created media query using font headline for title.